### PR TITLE
ceph: add volume replication crds

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

With the help of https://github.com/rook/rook/pull/7403/, rook will soon
integrate the volume replication controller as a side-car. Thus Rook
needs to inject the new CRDs.

Signed-off-by: Sébastien Han <seb@redhat.com>

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

**Review by umangachapagain — APPROVED** [review]

**Comment by leseb** [comment]
We are now waiting for a release tag to consume instead of sha1.

**Review by sp98 — APPROVED** [review]

File: cluster/examples/kubernetes/ceph/crds.yaml [hunk 1]
```diff
@@ -8284,6 +8284,200 @@ spec:
       subresources:
         status: {}
 
+---
```
Comment (travisn) [inline cluster/examples/kubernetes/ceph/crds.yaml:8287]: These CRDs aren't in the pre-1.16/crds.yaml. Is it safe to assume we require 1.16+ for the volume replication controller?

**Review by travisn — CHANGES_REQUESTED** [review]
LGTM, let's just wait to merge at the same time as #7403

**Comment by Madhu-1** [comment]
volume replication operator v0.1.0 release is available https://github.com/csi-addons/volume-replication-operator/releases/tag/v0.1.0

File: cluster/examples/kubernetes/ceph/crds.yaml (hunk 1 above)
Comment (leseb) [reply cluster/examples/kubernetes/ceph/crds.yaml:8287]: @Madhu-1 thoughts?

Comment (Madhu-1) [reply cluster/examples/kubernetes/ceph/crds.yaml:8287]: I think we can go with volume replication is supported only in the kubernetes 1.16+ version (technically it should work with older versions also but we have tested with the latest kubernetes versions.

**Review by travisn — APPROVED** [review]

**Pull request merged**